include/asm-generic/global_data.h

```diff
@@ -479,9 +479,11 @@ struct global_data {
 	 */
 	struct list_head dmtag_list;
 };
+#ifndef CONFIG_SEL4
```

Comment:
Whilst I don't have a counter-argument, it's not clear to me why we had to protect by adding `#ifndef CONFIG_SYS` here (and elsewhere).

Reply:
These are generally about limiting the subset of the U-Boot code that we pull in, or disentangling ourselves from the U-Boot build system. 

In this case the subset of code we pull in does not provide the GD_SIZE macro and pulling in the extra source files required to provide this macro did not seem to be a worthwhile trade-off.